### ISSUE TITLE
[BUG] Shrek 2 Menus on hardware renderer

### ISSUE THREAD

**Issue by seta-san** [PCSX2/pcsx2#4402]
**Describe the bug**
texture distortion in menu on hardware mode

**To Reproduce**
start game with hardware renderer

**Expected behavior**
menu doesn't distort

**GS Settings**
hardware renderer

**Emulation Settings**
defaults

**GS Window Screenshots**
https://www.youtube.com/watch?v=sh6K7ub2gxs&ab_channel=Seta-San

**System Info (please complete the following information):**
 -   PCSX2 Revision: 1.7.0-dev-1198
 -   OS: Windows 10
 -   CPU: i5-9600
 -   GPU: GTX 960

**Logs and Dumps**
[gsdx_20210426135950.zip](https://github.com/PCSX2/pcsx2/files/6379634/gsdx_20210426135950.zip)

**Comment by jeremybris** [comment]
I think this is part of #1318

**Comment by Hemporus01** [comment]
I think we have another issue to add to this one. It appears in SW mode as well. The second screenshot is SW.
![gs_20220319000749_Shrek 2 _NTSC-U__SLUS-20745](https://user-images.githubusercontent.com/97992262/159107771-f81bd5bd-c946-4ad9-bef2-38c4a9dc8a7a.png)
![gs_20220319001025_Shrek 2 _NTSC-U__SLUS-20745](https://user-images.githubusercontent.com/97992262/159107868-ca366bc1-f743-4faa-b8b3-820eadeadf17.png)
Also this message appears in the logs for some reason. Not sure if its the cause of the issue or not, but thought it might help.
<img width="359" alt="Capture" src="https://user-images.githubusercontent.com/97992262/159107983-e3102e55-6cb7-417c-9a07-74973f849531.PNG">

**Comment by refractionpcsx2** [comment]
if that message appears, it usually means you've ticked manual hw hacks in the Graphics settings, s it's disabled any fixes from the GameDB.

As for the software issue, are you going to tell us what the issue is, or do we have to guess?

**Comment by RedDevilus** [comment]
The orange message just says it will ignore renderer and upscaling fixes from the GameDB but Shrek 2 doesn't have any fixes be it clamping/rounding or otherwise.

**Comment by Hemporus01** [comment]
My bad forgot to upload the dump. Here goes
[shrek 2 white bloom.zip](https://github.com/PCSX2/pcsx2/files/8308647/shrek.2.white.bloom.zip)
.

**Comment by refractionpcsx2** [comment]
Right so the lighting cutoff weirdness at the back is the issue. Just making sure we're on the same page.

Was there anything else in the log? if you could upload the emulog after running the game at that point, it would be useful.

**Comment by Hemporus01** [comment]
Emulog? you mean like a blockdump?

**Comment by RedDevilus** [comment]
The file that contains program log / console contents.

**Comment by refractionpcsx2** [comment]
no I mean the emulog.txt file, from the logs folder, but only after you've run the game up to the point the problem appears. You need to run the actual game though, a GSDump won't contain the details we need.

**Comment by Hemporus01** [comment]
This issue is always present ingame no matter where you go. Here's the emulog
[emuLog.txt](https://github.com/PCSX2/pcsx2/files/8308668/emuLog.txt)
.

**Comment by refractionpcsx2** [comment]
Log doesn't really show anything of interest, shame :(

**Comment by Hemporus01** [comment]
I switched to software mode in the pause menu and this happened. I assume its related to the issue above
![gs_20220319003748_Shrek 2 _NTSC-U__SLUS-20745](https://user-images.githubusercontent.com/97992262/159108710-3c3000d0-a36c-47ec-af30-8948a0ec60fc.png)
.

**Comment by refractionpcsx2** [comment]
switching renderers when post effects are being done causes a lot of mess.

**Comment by Hemporus01** [comment]
I tried using half pixel offset and round sprite to see if they would get rid of the cutoff, but no dice. I'm not sure what the cause is.

**Comment by seta-san** [comment]
this is still a problem. in fact it may have gotten worse with a previous screen flashing in front of the menu and video

**Comment by seta-san** [comment]
never mind. it was fixed today. off by one testing
